packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -335,18 +344,19 @@ export class MirrorNodeClient {
         axiosRequestConfig['axios-retry'] = { retries };
       }
 
+      let response: AxiosResponse<T, any>;
       if (method === MirrorNodeClient.HTTP_GET) {
-        response = await this.restClient.get(path, axiosRequestConfig);
+        response = await this.restClient.get<T>(path, axiosRequestConfig);
       } else {
-        response = await this.web3Client.post(path, data, axiosRequestConfig);
+        response = await this.web3Client.post<T>(path, data, axiosRequestConfig);
       }
```

Comment:
We need to cover web3's `GET` request as well because the current implementation will not work against the mirror node's opcodes endpoint.

There are two consumable mirror node services by the relay:
- web3 API via `POST` method (used for `eth_call` and `eth_estimateGas`) running on local port :8545
- rest API via `GET` method (used for all other info queries) running on local port :5551